### PR TITLE
Remove poll from high level consumer destruct part 2

### PR DESCRIPTION
**Pull request by nick-zh**
#275

### CODE REVIEW

File: kafka_consumer.c [hunk 1]
```diff
@@ -50,16 +50,22 @@ static void kafka_consumer_free(zend_object *object TSRMLS_DC) /* {{{ */
 {
     object_intern *intern = get_custom_object(object_intern, object);
     rd_kafka_resp_err_t err;
+    kafka_conf_callbacks_dtor(&intern->cbs TSRMLS_CC);
 
     if (intern->rk) {
+        err = rd_kafka_assign(intern->rk, NULL);
```
Comment (arnaud-lb) [inline kafka_consumer.c]: This could be done in `kafka_conf_rebalance_cb`, when `cbs->rebalance` is NULL

**Comment by arnaud-lb** [comment]
Thanks !

**Comment by chasdevs** [comment]
This seems like just what I need to get proper failures in my environment. Will this be released any time soon?

**Comment by nick-zh** [comment]
yeah i hope so, in the meanwhile you could also build the master to get this fix

**Pull request merged**